core/api_backend.go

```diff
@@ -1,30 +1,30 @@
-package rpc
+package core
```

Comment:
I feel `api_backend` is only serving rpc, so putting it into rpc folder makes the code more cohesive.